draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -8113,33 +8113,43 @@ Content-Range: exampleunit 11.2-14.3/25
    method. For the methods defined by this specification, the intended meaning
    of the payload can be summarized as:
 </t>
-<dl>
-  <dt>GET</dt>
-  <dd>
-    a representation of the <x:ref>target resource</x:ref>;
-  </dd>
-  <dt>HEAD</dt>
-  <dd>
-    the same representation as GET, but without the representation data;
-  </dd>
-  <dt>POST</dt>
-  <dd>
-    a representation of the status of, or results obtained from, the action;
-  </dd>
-  <dt>PUT, DELETE</dt>
-  <dd>
-    a representation of the status of the action;
-  </dd>
-  <dt>OPTIONS</dt>
-  <dd>
-    a representation of the communications options;
-  </dd>
-  <dt>TRACE</dt>
-  <dd>
-    a representation of the request message as received by the
-    end server.
-  </dd>
-</dl>
+<table align="left">
+  <thead>
+    <tr>
+      <th>method</th>
+      <th>meaning of the payload</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>GET</td>
+      <td>a representation of the <x:ref>target resource</x:ref></td>
+    </tr>
+    <tr>
+      <td>HEAD</td>
+      <td>the same representation as GET, but without transferring the
+          representation data</td>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>a representation of the status of, or results obtained from,
+          the action</td>
+    </tr>
+    <tr>
+      <td>PUT, DELETE</td>
```

Comment:
Since it's a table, shouldn't these be in separate rows to aid scanning?

Reply:
Ha, we can use "rowspan" here :-)

Reply:
I tried it but the rowspan visualization doesn't work for both html and text. It just looks like a bug in the table.

I prefer the existing text to the alternative of repeating the content.